src/Inventory/Asset/Software.php

```diff
@@ -110,7 +110,7 @@ public function prepare(): array
 
             //If the software name exists and is defined
             if (property_exists($val, 'name') && $val->name != '') {
-                $val->name = trim(preg_replace('/\s+/', ' ', $val->name));
+                $val->name = trim(preg_replace('/\s+/u', ' ', $val->name));
```

Comment:
The problem with handling it this way is that software with the same name (but with a different GUID) merges

We can't handle ```<GUID>``` easily (straightening the data seems very complicated to me), maybe we can just keep an unbreakable space in DB (actually database stores it and returns it as a normal space).

Reply:
I agree the best solution would be to rely on the `GUID` to keep both software, but it's a new feature since they've always been merged. And also, software inventory is not simple, that new feature would take time to be implemented.